Model/Quote/QuoteIdMask.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Klaviyo\Reclaim\Model\Quote;
+use Magento\Framework\Model\ResourceModel\Db\AbstractDb;
+/**
+ * QuoteIdMask Resource model
+ * @codeCoverageIgnore
+ */
+class QuoteIdMask extends AbstractDb
```

Comment:
FYI The resource models in Magento2 projects are usually located in the `Model/ResourceModel/` folder.
It's better to move it eventually to prevent confusion.